src/Evolution/Systems/CurvedScalarWave/Worldtube/PunctureField.hpp

```diff
@@ -117,4 +117,32 @@ void acceleration_terms_0(
     const tnsr::I<double, 3>& particle_velocity,
     const tnsr::I<double, 3>& particle_acceleration, double ft, double fx,
     double fy, double dt_ft, double dt_fx, double dt_fy, double bh_mass);
+
+/*!
+ * \brief Computes the acceleration terms of a puncture/singular field
```

Comment:
Are these *only* the first order terms? Or do they include 0th order terms as well? Might be good to clarify if this function has to be called after `acceleration_terms_0` or if it's a standalone function.

Reply:
I added a small comment